pkg/utils/memory_resources.go

```diff
@@ -67,7 +67,12 @@ func GetNumaMemoryResources() (NumaMemoryResources, error) {
 		info[corev1.ResourceMemory] = nodeTotalMemory
 
 		// Get hugepages
-		hugepageBytes, err := getHugepagesBytes(filepath.Join(sysBusNodeBasepath, numaNode, "hugepages"))
```

Comment:
would it work if we just swallow IsNotExist here?
```
hugepageBytes, err := getHugepagesBytes(filepath.Join(sysBusNodeBasepath, numaNode, "hugepages"))
if err != nil && os.IsNotExist(err) {
    continue
}
```

Reply:
Good point 😅 I think it should work

Reply:
Ok, i will change it